app/models/user.rb

```diff
@@ -1,12 +1,22 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
   has_many :posts
   has_many :likes
   has_many :comments
+  after_initialize :initialize_post_counter
 
   validates :name, presence: true, length: { in: 5..30 }
   validates :posts_counter, presence: true, length: { in: 1..10 }, numericality: { only_integer: true }
 
   def user_posts
     Post.where('author_id = ?', id).limit(3)
   end
+
+  def initialize_post_counter
+    self.posts_counter = 0 unless posts_counter
+    self.photo = 'https://images.unsplash.com/photo-1560365163-3e8d64e762ef?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTB8fHNtaWxpbmclMjBmYWNlfGVufDB8fDB8fA%3D%3D&w=1000&q=80'
+  end
 end
```

Comment:
- 
![Capture](https://user-images.githubusercontent.com/63972670/216040771-7010a461-9214-461b-972f-d96cad4b8d0b.JPG)

Reply:
@OLIPLICHE Thank you very much for the review